docs/index.rst

```diff
@@ -1,15 +1,22 @@
-<PROJECT_NAME>
-==============================
+pytest asyncio network simulator
```

Comment:
This implies that you didn't use the script to fill in all the variables at the beginning. So there might be others of these template variables lingering around.

Here's the script which includes the template variables: https://github.com/carver/ethereum-python-project-template/blob/master/.project-template/fill_template_vars.sh#L40-L45